### PR TITLE
Document Azure AD Graph & Microsoft Graph permissions differences

### PR DESCRIPTION
**Pull request by FaithOmbongi**

### CODE REVIEW

File: concepts/migrate-azure-ad-graph-permissions-differences.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+author: "FaithOmbongi"
+ms.localizationpriority: medium
+ms.prod: "applications"
+---
+
+# Permissions differences between Azure AD Graph and Microsoft Graph
+
+This article is part of *step 1: review API differences* of the [process to migrate apps](migrate-azure-ad-graph-planning-checklist.md).
+
+An app that is granted Azure AD Graph permissions can only access Azure AD Graph resources. It can't access Microsoft Graph resources. For the app to access Microsoft Graph resources, you must grant it Microsoft Graph permissions.
```
Comment (Jackson-Woods) [inline concepts/migrate-azure-ad-graph-permissions-differences.md:13]: I'm not sure this is the right way to phrase it. Both Azure AD Graph and Microsoft Graph are accessing the same resources: same users, groups, etc. It's just that the means of accessing them is different.

Comment (Jackson-Woods) [inline concepts/migrate-azure-ad-graph-permissions-differences.md:13]: I'm not sure this is the right way to phrase it. Both Azure AD Graph and Microsoft Graph are accessing the same resources: same users, groups, etc. It's just that the means of accessing them is different.

File: concepts/migrate-azure-ad-graph-permissions-differences.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+
+## Directory.AccessAsUser.All
+
+### Delegated
+
+| Parameter               | Azure AD Graph                             | Microsoft Graph                            |
+|--------------------------|---------------------------------------------|---------------------------------------------|
+| Permission ID           | a42657d6-7f20-40e3-b6f0-cee03008a62a       | 0e263e50-5827-48a4-b97c-d940288653c7       |
+| Display String          | Access the directory as the signed-in user | Access the directory as the signed-in user |
+| Admin consent required? | No                                         | Yes                                        |
```
Comment (Jackson-Woods) [inline concepts/migrate-azure-ad-graph-permissions-differences.md:135]: Admin consent is required for Azure AD Graph as well.

**Review by Jackson-Woods — APPROVED** [review]
This looks great, thanks for compiling it. Some minor fixes below, first two comments the only ones I'd consider blocking.

File: concepts/migrate-azure-ad-graph-permissions-differences.md [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+### Application
+
+| Parameter      | Azure AD Graph | Microsoft Graph                      |
+|-----------------|-----------------|---------------------------------------|
+| Permission ID  | Not available  | df021288-bdef-4463-88db-98f22de89214 |
+| Display String | Not available  | Read all users' full profiles        |
+
+---
+
+## Enforce the principle of least privilege
```
[the remark this answers is not in the document]
Comment (FaithOmbongi) [reply concepts/migrate-azure-ad-graph-permissions-differences.md:301]: Makes sense; applying this.

File: concepts/migrate-azure-ad-graph-permissions-differences.md (hunk 1 above)
Comment (FaithOmbongi) [reply concepts/migrate-azure-ad-graph-permissions-differences.md:13]: Confirmed. Checking in with Philippe as well for a better/accurate intro.

**Pull request merged**